.github/workflows/ci.yml

```diff
@@ -107,13 +107,18 @@ jobs:
         os:
           - ubuntu-latest
           - [self-hosted, linux, arm64]
-          - macos-latest
-          - [self-hosted, macos, arm64]
+          - macos-12-large
+          - macos-13-xlarge
           - windows-latest
     steps:
       - name: Add OpenCL support - Ubuntu
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: sudo apt-get update -q && sudo apt-get install -qy ocl-icd-opencl-dev libpocl2
+
+      - name: Install coreutils
+        if: ${{ matrix.os == 'macos-12-large' || matrix.os == 'macos-13-xlarge' }}
+        run: brew install coreutils
```

Comment:
Why this additional step in the CI workflow?

Reply:
I didnt know if we would needed on the new runner. I'll remove. 1s

Reply:
```suggestion
```